src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2143,14 +2256,15 @@
 helps['network watcher packet-capture'] = """
     type: group
     short-summary: Manage packet capture sessions on VMs.
-    long-summary: |
-        These commands require that both Azure Network Watcher is enabled for the
-        VM's region and that AzureNetworkWatcherExtension is enabled on the VM.
+    long-summary: >
```

Comment:
Not clear why `|` was removed here. Newlines can be inserted with a double-newline when using this YAML flow. See https://stackoverflow.com/a/21699210/2562556 for the full, gory details on YAML text flowing.